src/hooks/slashtags2.ts

```diff
@@ -176,6 +176,13 @@ export const useMigrateSlashtags2 = (): void => {
 		}
 		status.current.profile = true;
 
-		saveProfile2(url, oldProfile, slashtagsProfile);
+		// save everything, except image becase it can be too big
+		const p = {
+			bio: oldProfile.bio,
+			name: oldProfile.name,
+			links: oldProfile.links,
+		};
+
+		saveProfile2(url, p, slashtagsProfile);
```

Comment:
Just a suggestion to make the code a bit cleaner and concise using destructuring. It's not a mandatory fix, just a tip! 😊

```suggestion
		// save everything, except image because it can be too big
		const { bio, name, links } = oldProfile;
		saveProfile2(url, { bio, name, links }, slashtagsProfile);
```